content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -314,6 +314,81 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Uninstall kubectl
+{{< tabs name="other_kubectl_uninstall" >}}
+
+{{% tab name="curl" %}}
+If you have used curl to install kubectl, then the uinstallation process can be done using :-
+
+```shell
+sudo rm -rf /usr/local/bin/kubectl
+sudo rm -rf /usr/local/lib/kubectl
+```
+
+An alternative command to carry out the uninstall:-
+
+```shell
+sudo rm $(which kubectl)
+```
+
+For getting binary files run the below command:
+
+```shell
+which kubectl
+```
+or
+```shell
+whereis kubectl
+```
+It is recommended to uninstall the executable binaries for completely removing the kubectl from the root.
+
+{{% /tab %}}
+
+{{% tab name="apt-get" %}}
```

Comment:
Just noticed that the previous [review comment](https://github.com/kubernetes/website/pull/41319#discussion_r1228828155) about the absence of steps with 'yum' package manager is still unresolved. Can you please address it in your future commits?

Reply:
Hi @SpringWiz11 Please address this comment here